components/menus/playnext.ts

```diff
@@ -21,11 +21,16 @@ import { generateUserCentric } from "../contracts/dataGen"
 import { controller } from "../controller"
 import type {
     GameVersion,
+    JwtData,
     MissionStory,
     PlayNextCampaignDetails,
+    UserCentricContract,
 } from "../types/types"
 
-export const orderedMissions: string[] = [
+/**
+ * Main story campaign ordered mission IDs.
+ */
+export const orderedMainCampaignMissions: string[] = [
```

Comment:
I feel we need like one file in the code base that has these kind of hardcoded lists, that way you will never had to scan the code and code only ever contains functions.